src/cswinrt/strings/WinRT.cs

```diff
@@ -1040,10 +1040,19 @@ protected override Delegate GetEventInvoke()
 
     // An event registration token table stores mappings from delegates to event tokens, in order to support
     // sourcing WinRT style events from managed code.
-    internal sealed class EventRegistrationTokenTable<T> where T : class, global::System.Delegate
+    internal sealed class EventRegistrationTokenTable<T>
+        where T : class, global::System.Delegate
```

Comment:
Is `where T : class` not implied from `where T : System.Delegate`? 😕

Reply:
I agree `class` looks unnecessary here.

Reply:
@jkotas For reference, there was some discussion on Discord looking into this. It was found that
* For any other base class constraint (aside from the special snowflake `System.Enum`), the compiler reports an error on the redundancy of the `class` constraint
* If you define `void M1<T>() where T : class, Delegate` and `void M2<T>() where T : Delegate`, [`M1` can be called from `M2` no problem](https://sharplab.io/#v2:EYLgxg9gTgpgtADwGwBYA0AXEBLANgHwAEAmARgFgAoKkgAkNIHYqBvK2j+lWgWVIB4AKgD4AFAEp2naQHcAFjFi1BtEPWJpaAERi4YAcwCGGGFI5tK0gL5UzXXsSFjJl6Z3mKYy1dt0Hjpq7mdtJ8ThIA3HY2lFZAA=)
  * Additionally, IL for `M2` contains `.param constraint` but it's not in `M1`. Not sure what difference that makes, but it does look weird that the technically *less* constrained method contains some kind of `constraint` instruction but the *more* constrained method does not.